src/shared/logger/commands.ts

```diff
@@ -43,6 +43,14 @@ export class Logging {
             return
         }
 
+        // HACK: The editor.document.getText() was returning a stale cached version of the logs,
+        // and due to this highlighting the new log would fail on subsequent calls of this function.T
+        // The following performs an action on the currently active file
+        // (which is why this command is right after the document is shown)
+        // and causes it to not use the cached getText() result.
+        // There is no guarantee for this behaviour in the future
```

Comment:
Using the past tense here makes the comment hard to understand. We can express the same idea with fewer words and more precision.

```suggestion
        // HACK: editor.document.getText() may return "stale" content,
        // then subsequent calls to openLogId() get confused.
        // Invoke "revert" on the current file to force vscode to read from disk.
```